storage/testonly/admin_storage_tester.go

```diff
@@ -165,14 +167,19 @@ func (tester *AdminStorageTester) TestCreateTree(t *testing.T) {
 			continue
 		}
 
-		// Make sure a tree was correctly stored
+		// Make sure a tree was correctly stored, and hasn't changed timestamps.
 		storedTree, err := getTree(ctx, s, newTree.TreeId)
 		if err != nil {
 			t.Errorf(":%v: getTree() = (%v, %v), want = (%v, nil)", test.desc, newTree.TreeId, err, newTree)
 			continue
 		}
-		wantTree = *storedTree
-		if !reflect.DeepEqual(newTree, &wantTree) {
+		if storedTree.CreateTimeMillisSinceEpoch != createTimeMillis {
+			t.Errorf("%v: getTree().CreateTime = %v; want %v", test.desc, storedTree.CreateTimeMillisSinceEpoch, createTimeMillis)
+		}
+		if storedTree.UpdateTimeMillisSinceEpoch != updateTimeMillis {
+			t.Errorf("%v: getTree().UpdateTime = %v; want %v", test.desc, storedTree.UpdateTimeMillisSinceEpoch, updateTimeMillis)
+		}
+		if !reflect.DeepEqual(storedTree, newTree) {
```

Comment:
Could you change this to use [pretty.Compare](https://godoc.org/github.com/kylelemons/godebug/pretty#example-Compare--Testing)? We get a diff with it, so we could remove the specialized create/update time checks too, I think.